guides/source/active_record_migrations.md

```diff
@@ -647,31 +797,55 @@ end
 ```
 
 INFO: Tables with composite primary keys require passing array values rather
-than integer IDs to many methods. See also the [Active Record Querying](
-active_record_querying.html) guide to learn more.
+than integer IDs to many methods. See also the [Active Record Composite Primary
+Keys](active_record_composite_primary_keys.html) guide to learn more.
 
-### When Helpers aren't Enough
+### Execute Arbitrary SQL
 
-If the helpers provided by Active Record aren't enough you can use the [`execute`][]
-method to execute arbitrary SQL:
+If the helpers provided by Active Record aren't enough, you can use the
+[`execute`][] method to execute arbitrary SQL. For example,
```

Comment:
I wonder if we can do better than "arbitrary" which has a negative connotation.

I think of this more as "SQL statements or commands that don't have Active Record helper methods". Using execute() is how we run that SQL. Although not sure how to say that succinctly.